panel/src/panel/drawer.test.js

```diff
@@ -11,7 +11,7 @@ describe.concurrent("panel.drawer", () => {
 			component: null,
 			isLoading: false,
 			isOpen: false,
-			island: true,
```

Comment:
While I like the new naming everywhere, I wonder if we should use something like legacy here instead.

Reply:
👍

Reply:
@bastianallgeier changed it